From: Robert Schwebel <rsc@pengutronix.de>
Subject: Makefile is trying to compile host tool with cross compiler

Singed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>

---
# mkl: added standard patch header, quilt refresh'ed
# wsa: 20091227: OE uses BUILD_CC here, buildroot doesn't have such a patch (why?)
#		 How to solve for upstream?

 lib/libcap/Makefile |    6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

Index: proftpd-1.3.2/lib/libcap/Makefile
===================================================================
--- proftpd-1.3.2.orig/lib/libcap/Makefile
+++ proftpd-1.3.2/lib/libcap/Makefile
@@ -9,6 +9,10 @@
 #
 topdir=$(shell pwd)/..
 include ../../Make.rules
+ifndef CC_FOR_BUILD
+CC_FOR_BUILD=$(CC)
+endif
+
 #
 # Library version
 #
@@ -26,7 +30,7 @@ OBJS=$(addsuffix .o, $(FILES))
 all: $(LIBNAME)
 
 _makenames: _makenames.c cap_names.sed
-	$(CC) $(CFLAGS) $(LDFLAGS) $< -o $@
+	$(CC_FOR_BUILD) $< -o $@
 
 cap_names.h: _makenames
 	./_makenames > cap_names.h
